code/extensions/che-api/src/impl/k8s-devworkspace-env-variables.ts

```diff
@@ -104,12 +97,6 @@ export class K8sDevWorkspaceEnvVariables {
       this.pluginRegistryURL = process.env.CHE_PLUGIN_REGISTRY_URL;
     }
 
-    if (process.env.CHE_PLUGIN_REGISTRY_INTERNAL_URL === undefined) {
```

Comment:
@svor @azatsarynnyy @vitaliy-guliy  
please let me know if we still should support the variable

Reply:
thank you!

Reply:
![giphy](https://github.com/che-incubator/che-code/assets/1271546/58c73a2f-f551-4437-84f3-73dab741ffc6)